windows/LiferayNativityShellExtensions/LiferayNativityContextMenus/LiferayNativityContextMenus.vcxproj

```diff
@@ -21,6 +21,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{E9BC5587-A688-4C36-8B15-DAFBAAAADCAE}</ProjectGuid>
     <RootNamespace>LiferayNativityContextMenus</RootNamespace>
```

Comment:
I thought about it, but I only wanted to change the public facing names with this PR. The namespace change affects literally every file in the project, and isn't worth the effort especially when there is a nonzero probability that we want to rebase liferays changes.